static/themes/default-runtime.css

```
@@ -177,11 +213,11 @@
   --color-neutral-quaternary: var(--color-gray-700);
   --color-neutral-quaternary-medium: var(--color-gray-600);
   --color-gray: var(--color-gray-600);
-  --color-brand-softer: var(--color-blue-950);
-  --color-brand-soft: var(--color-blue-900);
-  --color-brand: var(--color-blue-600);
-  --color-brand-medium: var(--color-blue-900);
-  --color-brand-strong: var(--color-blue-700);
+  --color-brand-softer: var(--color-primary-950);
```

Biome (2.1.2)

[error] Duplicate custom properties can lead to unexpected behavior and may override previous declarations unintentionally.

--color-brand-softer is already defined here.

Remove or rename the duplicate custom property to ensure consistent styling.

(lint/suspicious/noDuplicateCustomProperties)